tour-of-go/exports/databases/databases.go

```diff
@@ -0,0 +1,17 @@
+package databases
+
+import "fmt"
+
+var Data = "Data base"
+
+func DataBase() {
```

Comment:
According to what I taught you yesterday, correct this.

Function name must be a verb.


Please correct this in all packages.